cluster-autoscaler/config/autoscaling_options.go

```diff
@@ -164,6 +166,9 @@ type AutoscalingOptions struct {
 	// ScaleDownSimulationTimeout defines the maximum time that can be
 	// spent on scale down simulation.
 	ScaleDownSimulationTimeout time.Duration
+	// SchedulerConfig allows changing configuration of in-tree
+	// scheduler plugins acting on PreFilter and Filter extension points
+	SchedulerConfig *scheduler_config.KubeSchedulerConfiguration
```

Comment:
Let me think about this :pray:

Reply:
I guess the reason we want to keep it simple is because these are `AutoscalingOptions` and they should represent options instead of result of options.